### DISCUSSION TITLE
device/group binding

### DISCUSSION THREAD

**Discussion by timoline** [Koenkk/zigbee2mqtt#5451] [Devices]
Not all switch/remote devices can be bind to a device/group.... (I learned the hardway)
Maybe we should put this kind of info in  the support devices page?
https://www.zigbee2mqtt.io/information/supported_devices.html

**Comment by Koenkk** [comment]
I think this info should but put on the device page itself (it is already there for some devices). e.g to edit https://www.zigbee2mqtt.io/devices/D1.html press the link in: To contribute to this page, edit the following file

**Reply by timoline** [reply]
Any preference where on the device page?
seperate section?

**Reply by Koenkk** [reply]
Under notes, example: https://www.zigbee2mqtt.io/devices/E1524_E1810.html#binding

**Comment by doenietzomoeilijk** [comment]
Just to make sure and not create a new thread for nothing: this is a simple incompatibility between two different (probably brands of) devices, correct? So in my case, binding an IKEA dimmer to an IKEA bulb (or group if bulbs) works fine, but adding a Livarno bulb to that group doesn't (bulb gets added but doesn't respond to the dimmer).  Where would this be documented? At the dimmer or at the bulb? Or both?
